### PR TITLE
Prepare for release v3.7.0

### PR DESCRIPTION
**Pull request by ajfriend**

### CODE REVIEW

**Review by dfellis — APPROVED** [review]

**Comment by ajfriend** [comment]
I'm going through the instructions in `RELEASE.md`:

```
1. Create a PR "Preparing for release X.Y.Z" against master branch
    * Alter CHANGELOG.md from `[Unreleased]` to `[X.Y.Z] YYYY-MM-DD`
    * Run `make update-version` and give `X.Y.Z` when prompted
    * Check that all merges that need to be in the changelog are present
```

but I'm getting an error:

```
❯ make update-version
Scanning dependencies of target update-version
Current version: 3.7.0
Could not locate v3.7.0 as a Git revision.
```

Do I need to make a `v3.7.0` tag **locally** first? Or do I wait to do that on Github?

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +19,9 @@ The public API of this library consists of the functions declared in file
     - exactEdgeLengthRads
     - exactEdgeLengthKm
     - exactEdgeLengthM
+- refactor `getH3UnidirectionalEdgeBoundary` for accuracy at small resolutions (#391)
+    - speeds up `getH3UnidirectionalEdgeBoundary` by about 3x
+    - implement core logic for future vertex mode
```
Comment (nrabinowitz) [inline CHANGELOG.md:24]: Thanks for adding. Technically I guess this is a "fix", rather than a feature, but I think this is fine.

**Review by nrabinowitz — APPROVED** [review]

**Comment by dfellis** [comment]
> I'm going through the instructions in `RELEASE.md`:
> 
> ```
> 1. Create a PR "Preparing for release X.Y.Z" against master branch
>     * Alter CHANGELOG.md from `[Unreleased]` to `[X.Y.Z] YYYY-MM-DD`
>     * Run `make update-version` and give `X.Y.Z` when prompted
>     * Check that all merges that need to be in the changelog are present
> ```
> 
> but I'm getting an error:
> 
> ```
> ❯ make update-version
> Scanning dependencies of target update-version
> Current version: 3.7.0
> Could not locate v3.7.0 as a Git revision.
> ```
> 
> Do I need to make a `v3.7.0` tag **locally** first? Or do I wait to do that on Github?

We need to wait on @isaacbrodsky here, as I believe he has done most of the releases in the past? You're probably right about needing to run `git tag v3.7.0` yourself, but 1) why? 2) why is this not in the instructions? So would be best to confirm.

**Comment by isaacbrodsky** [comment]
> > I'm going through the instructions in `RELEASE.md`:
> > ```
> > 1. Create a PR "Preparing for release X.Y.Z" against master branch
> >     * Alter CHANGELOG.md from `[Unreleased]` to `[X.Y.Z] YYYY-MM-DD`
> >     * Run `make update-version` and give `X.Y.Z` when prompted
> >     * Check that all merges that need to be in the changelog are present
> > ```
> > 
> > 
> > but I'm getting an error:
> > ```
> > ❯ make update-version
> > Scanning dependencies of target update-version
> > Current version: 3.7.0
> > Could not locate v3.7.0 as a Git revision.
> > ```
> > 
> > 
> > Do I need to make a `v3.7.0` tag **locally** first? Or do I wait to do that on Github?
> 
> We need to wait on @isaacbrodsky here, as I believe he has done most of the releases in the past? You're probably right about needing to run `git tag v3.7.0` yourself, but 1) why? 2) why is this not in the instructions? So would be best to confirm.

The script expects the last published version. You will be prompted for the next version number after you specify the current version (probably should be read from `VERSION` instead of prompting).

**Review by isaacbrodsky — APPROVED** [review]

**Pull request merged**